subsys/nrf_security/src/drivers/cracen/cracenpsa/src/kmu.c

```diff
@@ -117,6 +117,16 @@ static psa_status_t get_encryption_key(const uint8_t *context, uint8_t *key)
 	return status;
 }
 
+static psa_status_t cracen_kmu_revoke_key_slot(int slot_id)
```

Comment:
this is static and it is also unused now, right?

Reply:
removed!